course_gather/settings/prod_settings.py

```diff
@@ -18,3 +25,17 @@
         'PORT': int(os.getenv('DB_PORT', '5432'))
     }
 }
+
+# Max-Age cache control
+WHITENOISE_MAX_AGE = 60 * 60
+
+# Cache database table
+CACHES = {
+    'default': {
+        'BACKEND': 'django.core.cache.backends.db.DatabaseCache',
+        'LOCATION': 'cache',
+    }
+}
+
+# Response cache time
+CACHE_MIDDLEWARE_SECONDS = 60 * 15
```

Comment:
I think we should make cache times the same (and also in seconds).
Currently, we have 3600 and 900.
Personally I think an hour (3600) should be fine, **if** this project ever generates a lot of traffic I can see potential issues with caching this long at the db level relating to size constraints. But we can always just switch over to memcached at that point.

Reply:
They're caching different things. I can add comments to make that more clear.

`CACHE_MIDDLEWARE_SECONDS` controls how long Django internally caches responses.

`WHITENOISE_MAX_AGE` controls how long the browser caches the resources it receives (just static files, not API responses). Since our static files have hashes appended to them during the build process, this value can (should) actually be near-infinite.

8e2af21.